modules/genomic_browser/js/profileColumnFormatter.js

```diff
@@ -1,66 +1,94 @@
+"use strict";
```

Comment:
@jstirling91 I guess the answer is yes according to the entry in the webpack.config.js file. But if you are asking, and the js file isn't minified maybe the answer is no then.